### PR TITLE
Feat new font kode mono

### PR DESCRIPTION
**Pull request by isaozler**
Hi David,

Saw my colleague @glennreyes ' twitter [post](https://twitter.com/glnnrys/status/1653417075329695761) about your awesome tool 🔥 and I wanted to add the developer typeface "Kode Mono" I recently released.

Kode Mono is a variable developer typeface with code-specific ligatures. Contains a weight variant of Regular, Medium, Semi-bold, and Bold.

It is currently in review by Google Font to be submitted. Link to the PR: https://github.com/google/fonts/pull/6239

https://www.kodemono.com is the webpage with some links and stuff

Hope you like it, love to get your feedback about this PR

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+:root {
+  --font-kode-mono: "Kode Mono";
+}
+
+@font-face {
+  font-family: "Kode Mono";
+  src: url("/fonts/KodeMono.woff2") format("woff supports variations"),
+       url("/fonts/KodeMono.woff2") format("woff-variations");
+  font-weight: 400 700;
+}
```
Comment (dragidavid) [inline styles/globals.css:14]: This here won't be necessary as we can use `localFont` from `next/font/local`

```suggestion

```

File: tailwind.config.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ module.exports = {
       inconsolata: ["var(--font-inconsolata)"],
       "source-code-pro": ["var(--font-source-code-pro)"],
       "ibm-plex-mono": ["var(--font-ibm-plex-mono)"],
+      "kode-mono": ["var(--font-mono-code)"],
```
Comment (dragidavid) [inline tailwind.config.js:26]:
```suggestion
      "kode-mono": ["var(--font-kode-mono)"],
```

**Review by dragidavid — CHANGES_REQUESTED** [review]
This is a really really nice font, thanks for requesting it! 

I just requested 2 changes here as we can use a consistent way to add new fonts.

There are 2 more files that would need to be changed for this to work, please see below:

Replace `lib/fonts.ts` with: https://shade.dragi.me/clh7pclpq0001kbnhzsmbfbw8
Replace `app/layout.tsx` with: https://shade.dragi.me/clh7pdeq80003kbnh175ss5lu

Thanks again for the great addition!

File: styles/globals.css (hunk 1 above)
Comment (isaozler) [reply styles/globals.css:14]: awesome 👍

File: tailwind.config.js (hunk 2 above)
Comment (isaozler) [reply tailwind.config.js:26]: my bad

**Comment by isaozler** [comment]
And again, what a fantastic tool you've crafted here!

**Review by dragidavid — APPROVED** [review]

**Pull request merged**